README.md

```diff
@@ -23,42 +23,38 @@ course use Slack at [tieto-comp-dev](https://tieto-comp-dev.slack.com/messages/)
        subject to change,
     1. You have two weeks since then to create a pull request with all
        the solutions,
-    1. ...but it doesn't there, we expect most of the learning to
-       happen during review process,
+    1. ...but it doesn't end there, we expect most of the learning to
+       happen during review process.
 1. We highly recommend solving the exercise just after the lesson,
 1. Don't hesitate to ask questions and don't take the reviews
    personally.
 
 We hope you will have a lot of fun learning!
 
 ### The Scope
-0. [Lesson 0 - The Setup](./course/lesson_00_the_setup/) -- deadline 2018-02-27
-1. Lesson 1 - Python basics
-1. Lesson 2 - Flow control
-1. Lesson 3 - Functions
-1. Lesson 4 - Unit testing
-1. Lesson 5 - Lists
-1. Lesson 6 - Dictionaries, tuples and sets
-1. Lesson 7 - \*args and \*\*kwargs
-1. Lesson 8 - Manipulating strings
-1. Lesson 9 - Regular expressions
-1. Lesson 10 - Reading and writing files
-1. Lesson 11 - Organizing Files
-1. Lesson 12 - argparse
-1. Lesson 13 - Debugging
-1. Lesson 14 - Keeping Time, Scheduling Tasks, and Launching Programs
-1. Lesson 15 - Objects and classes
+* [Lesson 0 - The Setup](./course/lesson_00_the_setup/) -- deadline 2018-02-27
```

Comment:
Czy faktycznie jest potrzebny deadline dla lekcji 0 i to tak długi?
Według mnie na lekcję 0 powinien pójść czas 13-18.02, w tym czasie powinni sobie skonfigurować środowisko, ewentualnie poprosić nas o pomoc, gdy mieli problem a od 19.02 ruszyć już z lekcją 1.

Reply:
Masz rację, ale jednak bym zostawił jak jest w imię prostoty. Generalnie mamy nadzięję, że uczestnicy będą robić możliwie do przodu.